packages/indexer-cli/src/rules.ts

```diff
@@ -137,7 +137,14 @@ export const parseIndexingRule = (
   const obj = {} as any
   for (const [key, value] of Object.entries(rule)) {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    obj[key] = (INDEXING_RULE_PARSERS as any)[key](value)
+    const parser = (INDEXING_RULE_PARSERS as any)[key]
+    if (!parser) {
+      throw (
+        `Indexing rule attribute '${key}' doesn't exist, please check spelling, might be one of these:\n` +
```

Comment:
What do you think about changing the text to this? 
`Indexing rule attribute '${key}' not supported, did you mean?`